src/functions.cpp

```diff
@@ -211,8 +213,16 @@ void make_movie(int num, vector<PARTICLE> &ion, INTERFACE &nanoparticle) {
                 type = "-1";
```

Comment:
@kadupitiya I was wondering if we should correct something here now. for some viewers, may be even ovito, -1 as a type may be a little hard to read -- should we call this type = "2". basically type 1 is positively charged ion, type 2 is negatively charged ion. type 2 is more accurate than type -1.

Reply:
unless of course, this type = -1 is used elsewhere in an important way and would need many other changes. i am ok with type = -1, but would prefer type 2 if that is a trivial change.